lib/ansible/modules/utilities/helper/_accelerate.py

```diff
@@ -518,7 +518,7 @@ def fetch(self, data):
             fd.close()
             tb = traceback.format_exc()
             log("failed to fetch the file: %s" % tb)
-            return dict(failed=True, stderr="Could not fetch the file: %s" % str(e))
```

Comment:
Do you mean that when Python 2 is used, `e` could be an exception raised by a third party library (not by the python standard library) and third party libraries aren't trusted to raise exceptions containing unicode message ?